nuxt/pages/moderator/DraftLyricsPage.vue

```diff
@@ -0,0 +1,99 @@
+<router>
+  path: /moderator/drafts/lyrics
+  name: "moderator.drafts.lyrics"
+</router>
+
+<template>
+  <v-container class="app__section mt-4">
+    <h2>Draft Lyrics</h2>
+    <div v-if="draftLyrics.length > 0" class="draft-lyrics">
+      <v-overlay v-if="$fetchState.pending" absolute class="revisions__loading">
+        <v-progress-circular
+          indeterminate
+        />
+      </v-overlay>
+      <draft-lyrics-card
+        v-for="draft in draftLyrics"
+        :key="draft.id"
+        :draft-lyric="draft"
+      />
+      <v-pagination
+        v-model="page"
+        class="my-8"
+        color="deep-orange"
+        :length="length"
+        :total-visible="10"
+        circle
+        next-icon="navigate_next"
+        prev-icon="navigate_before"
+        @input="onPageChanged"
+      />
+    </div>
+    <div v-else-if="$fetchState.pending" class="revisions__loading text-center">
+      <v-progress-circular indeterminate />
+    </div>
+    <div v-else class="revisions__empty">
```

Comment:
We also have an empty state component that we could use here

Reply:
Already added the empty state here
```
<div v-else class="draft-lyrics__empty">
      There's nothing here.
    </div>
```

Same as revisions